frontend/src/api/prices/element.ts

```diff
@@ -0,0 +1,22 @@
+import { ELEMENT_SPECIFIC_TOKENS } from '@/constants/apy-mainnet-constants'
+import { BigNumber, ethers, getDefaultProvider } from "ethers";
+import { WAD } from "@/constants/static";
+
+export const isElementSpecificToken = (baseTokenNameLowercase: string) => (
+  ELEMENT_SPECIFIC_TOKENS.includes(baseTokenNameLowercase)
+)
+
+// written to use this contract: https://etherscan.io/address/0x557f483e13c94a02cbfefe1d27a781c4221f37f5#readContract
+export const getPriceValueFromContract = async (
+  contractAddress: string,
+): Promise<number> =>  {
+  console.log("getPriceValueFromContract")
+  if (contractAddress) {
```

Comment:
if the type is string is this necessary? Are you expecting an empty string potentially?

Reply:
fair point